pkgs/by-name/to/tocaia/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  stdenv,
+  fetchgit,
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Lets use `finalAttrs` which is a nixpkgs pattern that avoid `rec` pitfalls:
- #315337